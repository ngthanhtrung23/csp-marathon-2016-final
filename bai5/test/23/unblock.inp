.12256
.13456
003459
.778.9
...8..
.AABB.
