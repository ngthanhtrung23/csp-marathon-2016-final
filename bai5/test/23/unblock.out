26
5 D 1
7 L 1
9 D 1
6 D 1
2 R 2
4 U 1
A L 1
3 D 3
0 R 2
B R 1
8 D 1
7 R 2
1 D 3
0 L 1
4 D 1
2 L 3
4 U 1
0 R 1
1 U 2
6 U 1
7 L 2
3 U 1
8 U 1
B L 2
5 D 2
0 R 2

